app/Http/Controllers/Legacy/Two/EventController.php

```diff
@@ -4,6 +4,7 @@
 
 use Rogue\Models\Event;
 use Illuminate\Http\Request;
+use Rogue\Http\Controllers\ApiController;
```

Comment:
Above it looks like you moved `Legacy/Two/ApiController` to just `/ApiController` and then here you use the new `/ApiController` in a v2 API route. I'm wondering why we wouldn't be using `Legacy/Two/ApiController` in `/Legacy/Two/EventController.php`?

Reply:
hm you're right - I didn't notice that actually! Thanks for pointing that out, @sbsmith86 !

The only different between the `Event` and `Events` Controllers is the different middleware. I think it makes sense to delete the `/Legacy/Two/EventController.php`and leave the `Events` Controller and have everything that used `/Legacy/Two/EventController.php` to use the `Events` Controller at the root of the Controller directory?

Reply:
I think that anything that lives under `Legacy/Two` should use things that live under `Legacy/Two` 

I think that will make it easier to kill because it will really be silo'ed.

For example, here we are using root level `ApiController` in a `Legacy/Two` classes so that starts to get confusing to follow.

Reply:
that makes sense to me too! The reason why I moved `Legacy/Two/ApiController` to the root level of the Controllers is because v3 controllers also use `Legacy/Two/ApiController`. I thought it made more sense for this shared file to live at the root so when we do get rid of the `Legacy/Two` classes, we don't have to move the `ApiController` then. 

Is this ok for that reason or should I create two separate `ApiControllers`, similar to the `Events/Event` Controllers that live both under the Legacy folder and root folders?

Reply:
I see, that makes sense. I think my preference would be to have a `Legacy/Two/ApiController` and a `/ApiController`.

v2 stuff would use  `Legacy/Two/ApiController`
v3 stuff would use `/ApiController` 

This way any new development could happen in `/ApiController` without v2 stuff being affected.

Reply:
ah yes that makes a ton of sense to do it that way - thanks @sbsmith86 ! I'll update now :)